spec/presenters/tree_builder_ops_rbac_features_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe TreeBuilderOpsRbacFeatures do
+  let(:features) do
+    %w(
+      all_vm_rules
+      instance
+      instance_view
+      instance_show_list
+      instance_control
+      instance_scan
+    )
+  end
+
+  let(:role) do
+    FactoryGirl.create(:miq_user_role, :id => 100_000_002, :features => features)
+  end
+
+  let(:tree) do
+    TreeBuilderOpsRbacFeatures.new(
+      "features_tree",
+      "features",
+      {:trees => {}},
```

Comment:
But it is in a `let`? If I understand what you're saying, even Rubocop disagrees with you on the readability.